styles.css

```diff
@@ -712,9 +712,120 @@ body {
   hr {
     display: none;
   }
+
+  .nav-link {
+    text-decoration: none;
+    transition: 0.7s ease;
+    color: #3a4a42;
+  }
+
+  .nav-item {
+    margin: 16px 0;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 32px;
+    line-height: 44px;
+    display: flex;
+    align-items: center;
+    color: #3a4a42;
+  }
+
+  .nav-menu.active {
+    left: 0;
+  }
```

Comment:
- [OPTIONAL] Great job here 💯 However, I suggest adding `bottom: 0` to `.nav-menu.active` selector to make the menu fill all the height 👍

![image](https://user-images.githubusercontent.com/45231276/195256505-091623cd-601c-4e07-97bd-14e315a19ff0.png)